modules/document-db/database-account/gremlin-database/main.bicep

```diff
@@ -60,10 +62,7 @@ resource gremlinDatabase 'Microsoft.DocumentDB/databaseAccounts/gremlinDatabases
   name: name
   tags: tags
   parent: databaseAccount
-  identity: (identityType != 'None' ? {
-    type: identityType
-    userAssignedIdentities: !empty(userAssignedIdentities) ? userAssignedIdentities : null
-  } : null)!
+  identity: identity!
```

Comment:
Same as in the other case, is `!` a good idea here?

Reply:
It looks like there is no identity support for the Gremlin Database, neither in the documentation nor in the module (this parameter is not passed on to the child resource) nor in the DB settings in the portal. This means that the warning is legit. I've removed the identity support from the child-resource. (commit: https://github.com/Azure/ResourceModules/pull/4124/commits/7dbf049f8b6217fc58fec4094e3c8e68d10b8c78)